Add new example workspace reading an `.RData` file and plotting from included `data.frame`

Timmimim opened the pull request:
New workspace includes an `.RData` file. It contains the `data.frames` named `iris` and `mtcars`, which are both included in the `R` language. 

This data file is read on execution by the workspace's `main.Rmd` file. The RMarkdown script creates a small set of example plots from the data, and outputs those as a mock HTML paper.

nuest commented:
Please change the name of the datasets, because `iris` and `mtcars` are available in any R session without loading any data. You could just assign new names and save them to the data file, e.g. `my_iris <- iris; plot(my_iris ...)` etc.

![image](https://user-images.githubusercontent.com/1325054/41597664-61849fd0-73ce-11e8-8181-288d95c67c82.png)

Please also add an un-executed cell in the `Rmd` file with the code for creating the data file.

Timmimim commented:
Changed object names, updated `example:RData`;
Added code chunks generating `.RData` file to `main.Rmd`, excluded it from execution;
Updated `main.Rmd` to work with new object names `my_iris` and  `my_mtcars`;
Generated new `display.html` file;  commited changes:

https://github.com/o2r-project/erc-examples/pull/1/commits/8b0be163d396968a1cc6f35aa987e411aa056b3f
